.cirrus.yml

```diff
@@ -247,13 +247,19 @@ task:
         matrix:
           CHANNEL: "master"
           CHANNEL: "stable"
+        CHROME_NO_SANDBOX: true
+        CHROME_DIR: /tmp/web_chromium/
+        CHROME_EXECUTABLE: $CHROME_DIR/chrome-linux/chrome
       install_script:
+        - ./script/install_chromium.sh "$CHROME_DIR"
         - git clone https://github.com/flutter/web_installers.git
         - cd web_installers/packages/web_drivers/
         - dart pub get
       chromedriver_background_script:
         - cd web_installers/packages/web_drivers/
-        - dart lib/web_driver_installer.dart chromedriver --install-only
+        # web_driver_installer only handles Google Chrome, not Chromium, so
+        # specify a version to match what's installed by install_chromium.sh
+        - dart lib/web_driver_installer.dart chromedriver --install-only --driver-version=84.0.4147.30
```

Comment:
How much does the `web_installers` package buy you? You could install the driver together with Chromium in the same script (probably just a couple of lines of bash). This way when rolling Chromium and chromedriver to a new version you only need to change one file, and there's one fewer dependencies on an external package (also, secretly, I'd like to eventually delete `web_installers` altogether).

Reply:
Good call; I didn't even consider doing it directly since I'm used to `web_installers` being The Way To Do It. I replaced it with a direct wget/unzip in `install_chromium.sh`.